ansible/scripts/restart-master.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+. ./init.sh
+
+inventory=${INVENTORY:-${INVENTORY_DIR}/inventory}
+ansible-playbook -i ${inventory} ${PLAYBOOKS_DIR}/deploy-master.yml \
+		--tags "restart" \
```

Comment:
I wanted to align all options as close to each other as possible. What about

```
ansible-playbook \
  -i ${inventory} ${PLAYBOOKS_DIR}/deploy-master.yml \
  --tags "restart" \
  --extra-vars "apiserver_modified=true controller_manager_modified=true scheduler_modified=true kubelet_modified=true" \
  "$@"
```

?